packages/jwt-verifier/test/util.js

```diff
@@ -27,18 +27,21 @@ function getAccessToken(options = {}) {
     const urlProperties = url.parse(ISSUER);
     const domain = urlProperties.protocol + '//' + urlProperties.host;
     const postUrl = domain + '/api/v1/authn';
-    request.post(postUrl, {
-      json: true,
-      body: {
+
+    fetch(postUrl, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
         username: USERNAME,
         password: PASSWORD
+      })
+    }).then(resp => {
+      if (!resp.ok) {
+        throw new Error(`/api/v1/authn returned error: ${resp.status}`);
       }
-    }, function (err, resp, body) {
-      const statusCode = resp ? resp.statusCode : null;
-      if (err || (resp && resp.statusCode >= 400) || !resp) {
-        console.error('/api/v1/authn returned error: ', statusCode, err || body);
-        return reject(new Error('/api/v1/authn error: ' + (err || body)));
-      }
+
+      return resp.json();
+    }).then(body => {
```

Comment:
Possibly worth considering. Some errors will be returned with a 200 response code. In this case there will be no sessionToken and there should be an error code / error summary.  See how SIW handles this: https://github.com/okta/okta-signin-widget/blob/master/src/util/Util.js#L79  We might want to avoid making the next request if there is no sessionToken, and instead throw an error. 

It is also possible (though less likely) that the server will return a 200 status code but invalid JSON, in which case `resp.json()` will throw. This case I think we can leave unhandled because it will already throw. But we are handling it in AuthJS 3.0 (for the sake of better errors in the SIW): https://github.com/okta/okta-auth-js/pull/326/files#diff-ed946c93045cf8b90899ada2e4655327R21

Reply:
Good point. As the change is only for a util function in test, i would propose to handle empty `sessionToken`, then leave invalid json to `catch`.

Reply:
Yeah it would be only to help us with better error message in the test logs.